data/rdr3/scenarios.lua

```diff
@@ -2064,26 +3166,41 @@ Scenarios = {
 	"WORLD_PLAYER_PICKUP_WEAPON_RACK_DBSHOTGUN",
 	"WORLD_PLAYER_PICKUP_WEAPON_RACK_DBSHOTGUN_WORN",
 	"WORLD_PLAYER_PICKUP_WEAPON_THROWN_TOMAHAWK_ANCIENT",
+	"WORLD_PLAYER_PICK_LOCK",
 	"WORLD_PLAYER_PLACE_BAIT",
+	"WORLD_PLAYER_PLACE_BAIT_NORMAL",
 	"WORLD_PLAYER_PLACE_BAIT_POTENT",
 	"WORLD_PLAYER_PRPTY_FAST_TRAVEL",
 	"WORLD_PLAYER_RANSACK_PICKUP_MONEYBAG_FRONT_TAKE_LEFT_SHOULDER",
+	"WORLD_PLAYER_RANSACK_PICKUP_MONEYBAG_FRONT_TAKE_RIGHT_SHOULDER",
+	"WORLD_PLAYER_RE_ANIMAL_MAULING",
+	"WORLD_PLAYER_RE_DARK_ALLEY_DESP_GIVE",
+	"WORLD_PLAYER_RE_DARK_ALLEY_DRUNK_HELP",
+	"WORLD_PLAYER_RE_DARK_ALLEY_DRUNK_STEAL",
+	"WORLD_PLAYER_RE_LOST_DOG_FEED",
 	"WORLD_PLAYER_RHO_GUN_BSMT_PEEK",
 	"WORLD_PLAYER_RIFLE_PICKUP_LEAN",
+	"WORLD_PLAYER_RSC_GIVE_MEDICINE",
 	"WORLD_PLAYER_SEAT_LEDGE",
 	"WORLD_PLAYER_SEAT_LEDGE_HIDDEN",
 	"WORLD_PLAYER_SHOPPING",
+	"WORLD_PLAYER_SIT_GROUND_GENERIC",
 	"WORLD_PLAYER_SLEEP_BEDROLL",
 	"WORLD_PLAYER_SLEEP_GROUND",
 	"WORLD_PLAYER_STR_GEN_BSMT_PEEK",
 	"WORLD_PLAYER_TAKE_MONEY_PILE_TABLE",
+	"WORLD_RIDER_ATTRACTOR",
 	"WORLD_VEHICLE_ATTRACTOR",
 	"WORLD_VEHICLE_DRIVE_PASSENGERS",
+	"WORLD_VEHICLE_DRIVE_PASSENGERS_NO_RAIN",
 	"WORLD_VEHICLE_DRIVE_SOLO",
+	"WORLD_VEHICLE_DRIVE_SOLO_NO_RAIN",
 	"WORLD_VEHICLE_EMPTY",
 	"WORLD_VEHICLE_EMPTY_NO_ANIMALS",
 	"WORLD_VEHICLE_MINECART_COAL_DUMP",
 	"WORLD_VEHICLE_MINECART_COAL_DUMP_MALE",
 	"WORLD_VEHICLE_MINECART_COAL_SHOVEL",
+	"WORLD_VEHICLE_MINECART_COAL_SHOVEL_MAL",
 	"WORLD_VEHICLE_MINECART_COAL_SHOVEL_MALE",
+	"sssssssssssssss",
```

Comment:
Thanks! Just one question though, is this actually a valid scenario?

Reply:
Not valid scenario. My mistake. Fixed

Reply:
Awesome, thanks for checking!